src/cyhy_cvesync/cve_sync.py

```diff
@@ -20,6 +20,7 @@
 ALLOWED_URL_SCHEMES = ["http", "https"]
 CVE_URL_RETRY_WAIT_SEC = 5
 MAX_CVE_URL_RETRIES = 10
+PREFERRED_CVSS_METRICS = ["cvssMetricV31", "cvssMetricV30", "cvssMetricV2"]
```

Comment:
This should probably get a quick comment that this is in order of preference/priority.